tests/bootstrap.php

```diff
@@ -4,7 +4,7 @@
 
 namespace Mathematicator\Calculator\Tests;
 
-// TODO: require __DIR__ . '/../vendor/autoload.php';
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
This path does not exist in case of you install package to Nette project.

Reply:
tests/ folder is intended only for dev environment in package scope. Tests should be run only on package scope.

Reply:
@janbarasek Please see this section in package.json:
```
	"autoload": {
		"classmap": [
			"src/"
		]
	},
	"autoload-dev": {
		"classmap": [
			"tests/"
		]
	},
```

Reply:
The test file can be loaded by the autoloader when the application is initialized. In this case, a fatal error is thrown.

Can you ensure that the autoloader is loaded only when the test is running?

Reply:
Yes, I am 100% sure. For sure, I have just tried it just now and it is as I said. This is common solution for tests. The problem is with your Doctrine setup, as we have discussed before, you should map Doctrine only to concrete folders e.g. src/Entity in vendor pack...